compiler/src/dmd/cxxfrontend.d

```diff
@@ -265,10 +265,16 @@ bool functionSemantic3(FuncDeclaration fd)
     return dmd.funcsem.functionSemantic3(fd);
 }
 
-MATCH leastAsSpecialized(FuncDeclaration f, FuncDeclaration g, Identifiers* names)
+MATCH leastAsSpecialized(FuncDeclaration fd, FuncDeclaration g, Identifiers* names)
 {
     import dmd.funcsem;
-    return dmd.funcsem.leastAsSpecialized(f, g, names);
+    return dmd.funcsem.leastAsSpecialized(fd, g, names);
+}
+
+PURE isPure(FuncDeclaration fd)
+{
+    import dmd.funcsem;
+    return dmd.funcsem.isPure(fd);
 }
```

Comment:
I don't think we need to expose this to C++, it is unused by LDC

Reply:
@thewilsonator thanks. Any way I can check before I make furure PRs?

Reply:
I use `git grep` on my local clone of LDC. I guess you could also use GitHub search and use the language filter and see if you get any hits of C++ files.

Reply:
LDC isn't the only compiler, so don't make presumptions from one data point. ;-)

Reply:
for codegen functions yes, but for semantic functions its a pretty good barometer